Application/EdFi.Ods.Api/Models/AssemblyData.cs

```diff
@@ -0,0 +1,25 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+
+namespace EdFi.Ods.Api.Models
+{
+    public class AssemblyData
```

Comment:
This class does not appear to be used and looks maybe like it was just copied in from the CodeGen utility.

Please remove it.

Reply:
Fixed it . Thanks